docs/events.md

```diff
@@ -68,3 +68,129 @@ Resource      |Event    |Event Type
 `PipelineRun` | `Condition Change while Running` | `dev.tekton.event.pipelinerun.unknown.v1`
 `PipelineRun` | `Succeed` | `dev.tekton.event.pipelinerun.successful.v1`
 `PipelineRun` | `Failed`  | `dev.tekton.event.pipelinerun.failed.v1`
+
+## Format of `CloudEvents`
+
+According to the `CloudEvents` spec, HTTP headers are included to match the context fields. For example:
```

Comment:
Maybe add a link to the spec? https://github.com/cloudevents/spec/blob/master/spec.md

Reply:
Thanks! Done